github/projects.go

```diff
@@ -261,12 +261,15 @@ func (s *ProjectsService) MoveProjectColumn(ctx context.Context, columnID int, o
 //
 // GitHub API docs: https://developer.github.com/v3/repos/projects/
```

Comment:
This GitHub API docs link is pretty generic and out of date (the URL has changed). Mind updating it to something more relevant, like https://developer.github.com/v3/projects/cards/#get-a-project-card?